src/js/models/messages.js

```diff
@@ -44,4 +44,19 @@ export default class Messages{
 			alert(error);
 		}
 	}
+
+	async getSearchResults(searchContent) {
+		let arr = [];
+		try{
+			await db.collection("Profiles").where("name", "==", searchContent).get().then(function(querySnapshot) {
+        		querySnapshot.forEach(function(doc) {
+            		arr.push(doc.data());
+				});
+			});
+			this.searchRes = arr;
```

Comment:
correct indent lines 51 to 56?

Reply:
Suggested change at https://github.com/Yuxuan-Lin/VioLegacy/pull/31#pullrequestreview-509961094